src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(() -> new DataProcessingException(
+                "Can't find car by id: " + id + "."));
```

Comment:
```suggestion
        return carDao.get(id).orElseThrow(() -> new RuntimeException(
                "Can't find car by id: " + id + "."));
```
i think here you must use RE, DPE we must use for DAO

Reply:
Я делал так в прошлом задании. Специально добавлял конструктор с одним параметром. В этом уже два конструктора.